build/tools/ExtractRuleDescFromJson/RuleDescExtractor.cs

```diff
@@ -111,7 +111,9 @@ private void ProcessRule(PluginRule pluginRule)
                 ConvertPluginSeverity(pluginRule.DefaultSeverity),
                 ConvertPluginIssueType(pluginRule.Type),
                 Convert.ToBoolean(pluginRule.IsActiveByDefault),
-                pluginRule.Tags ?? Array.Empty<string>()
+                pluginRule.Tags ?? Array.Empty<string>(),
+                null,
```

Comment:
for now it's null to avoid compile errors. It should be implemented with extraction task.